### PR TITLE
[exporter/signalfxexporter] exclude wmi v1.2.0

### LINKED ISSUE #4433: checkSum failed at the current main HEAD
**Describe the bug**
For the current main HEAD, at the project root

```
go mod -tidy
....
verifying github.com/StackExchange/wmi@v1.2.0/go.mod: checksum mismatch
	downloaded: h1:rcmrprowKIVzvc+NUiLncP2uuArMWLCbu9SBzvHz7e8=
	go.sum:     h1:3eOhrUMpNV+6aFIbp5/iudMxNCF27Vw2OZgy4xEx0Fg=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.

```

**Steps to reproduce**
Even if I remove go.sum and run go clean -modcache, and then rerun `go mod -tidy`, it results in

```
verifying github.com/StackExchange/wmi@v1.2.0/go.mod: checksum mismatch
	downloaded: h1:rcmrprowKIVzvc+NUiLncP2uuArMWLCbu9SBzvHz7e8=
	sum.golang.org: h1:3eOhrUMpNV+6aFIbp5/iudMxNCF27Vw2OZgy4xEx0Fg=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.
```

**What did you expect to see?**
go mod -tidy should succeed

**What did you see instead?**
It failed

**What version did you use?**
current main HEAD

**What config did you use?**
Config: (e.g. the yaml config file)

**Environment**
OS: MacOS mojav 10.14.6
Compiler(if manually compiled): go1.13.4

```
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/qchea/Library/Caches/go-build"
GOENV="/Users/qchea/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GONOPROXY="github.com/chenqi0805"
GONOSUMDB="github.com/chenqi0805"
GOOS="darwin"
GOPATH="/Users/qchea/go"
GOPRIVATE="github.com/chenqi0805"
GOPROXY="direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/darwin_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/Users/qchea/go/src/opentelemetry-collector-contrib/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/jm/02gfdshn1vx7mrk429ksj7bw6h09xs/T/go-build226447730=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

**Additional context**

### PR DESCRIPTION
**Pull request by mx-psi**
**Description:** <Describe what has changed.>

Add `exclude` directive for `github.com/StackExchange/wmi` version `1.2.0`

**Link to tracking Issue:** fixes #4433

### CODE REVIEW

**Comment by anuraaga** [comment]
@mx-psi I checked out this branch but unfortunately still get the same error. Perhaps the same has to be added to the top level go.mod?

```
verifying github.com/StackExchange/wmi@v1.2.0/go.mod: checksum mismatch
        downloaded: h1:rcmrprowKIVzvc+NUiLncP2uuArMWLCbu9SBzvHz7e8=
        go.sum:     h1:3eOhrUMpNV+6aFIbp5/iudMxNCF27Vw2OZgy4xEx0Fg=
```

**Comment by anuraaga** [comment]
Nope just tried it and still didn't work. Only `replace` seems to work

File: go.mod [hunk 1]
```diff
@@ -76,7 +76,6 @@ require (
 	github.com/stretchr/testify v1.7.0
 	go.opentelemetry.io/collector v0.31.1-0.20210810171211-8038673eba9e
 	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c
-	golang.org/x/tools v0.1.5 // indirect
```
Comment (mx-psi) [inline go.mod:79]: Result of doing `go mod tidy`

**Review by anuraaga — APPROVED** [review]
Thanks @mx-psi!

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**